lib/bloc/market_maker_bot/market_maker_trade_form/market_maker_trade_form_state.dart

```diff
@@ -93,6 +97,15 @@ class MarketMakerTradeFormState extends Equatable with FormzMixin {
   /// This value is fetched from the DEX API and cached in the state.
   final Rational? maxMakerVolume;
 
+  /// The minimum trading volume required by the base coin (sell coin).
+  /// Retrieved from the DEX API `min_trading_vol` for clearer validation
+  /// and error messaging.
+  final Rational? minTradingVolume;
```

Comment:
nitpick: rename `minTradingVolume` to avoid confusion with existing `minimumTradeVolume` input field state.

```suggestion
  final Rational? minimumBaseVolume;
```